apps/devhub/views.py

```diff
@@ -684,6 +684,12 @@ def upload_for_addon(request, addon_id, addon):
     return upload(request, addon_slug=addon.slug)
 
 
+@post_required
+@dev_required
+def upload_for_addon_unlisted(request, addon_id, addon):
+    return upload(request, addon_slug=addon.slug, is_listed=False)
```

Comment:
It's the view that calls the validator. If one would forge it, it would simply result in a run of the validator with the "not on amo" parameter, meaning it checks/allows a `updateUrl` entry in the xpi.

I could add the check, but really it seems useless, what do you think?